ecmascript/transforms/src/helpers/mod.rs

```diff
@@ -230,14 +266,61 @@ impl Fold<Span> for Marker {
 
 #[cfg(test)]
 mod tests {
-    use super::InjectHelpers;
+    use super::*;
+
+    #[test]
+    fn external_helper() {
+        let input = "_throw()
+swcHelpers._throw()";
+        crate::tests::Tester::run(|tester| {
+            HELPERS.set(&Helpers::new(true), || {
+                let expected = tester.apply_transform(
+                    ::testing::DropSpan,
+                    "output.js",
+                    Default::default(),
+                    "import * as swcHelpers1 from '@swc/helpers';
```

Comment:
Importing them with a star means potentially bundling a lot of unused helpers. Having them named helps save some bytes.

It really is a two edged sword, like sometimes the whole re-importing text written dozens of times amounts to more space than caused by the unused helpers. Really depends on size of helpers and how much you think in a near-enough-to-worry-about future.

Babel went towards the named individually path, do you think it makes sense for swc to adopt the same pattern?

Reply:
Babel's helper works in this way and module bundler with tree shaking can remove unused imports. So I'm not sure importing only used stuffs is necessary.

Reply:
I believe if '@swc/helpers' itself can be statically analyzed (only using es6 imports/exports) a bundler can do the needed tree shaking.  

Although I don't actually know which bundlers do and which don't do this...